### PR TITLE
Added dist and targets

### PR DESCRIPTION
**Pull request by aledomu**
The build-electron per-platform scripts were removed, since with the new dist scripts they become unnecessary.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,37 +1,65 @@
 {
   "name": "mustard",
   "version": "0.0.1",
-  "description": "A tagger tool for music files and simpe player",
+  "description": "A tagger tool for music files and simple player",
```
Comment (angrykoala) [inline package.json:4]: good spot

File: package.json [hunk 2]
```diff
@@ -1,37 +1,65 @@
...
+      ],
+      "category": "public.app-category.music"
+    }
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/angrykoala/mustard.git"
   },
-  "author": "angrykoala <demiurgosoft@hotmail.com>",
+  "author": "angrykoala <angrykoala@outlook.es>",
```
Comment (angrykoala) [inline package.json:55]: nice, thanks

File: package.json [hunk 3]
```diff
@@ -1,37 +1,65 @@
...
   "scripts": {
     "start": "electron .",
     "test": "echo \"Error: no test specified\" && exit 1",
     "jshint": "jshint index.js app/*.js",
     "build-electron": "electron-packager . --overwrite --out builds --ignore='\\.gitignore|\\.jshintrc'",
     "build-electron-all": "npm run build-electron-win && npm run build-electron-linux && npm run build-electron-mac",
-    "build-electron-win": "electron-packager . --overwrite --out builds --platform=win32 --arch=x64",
-    "build-electron-linux": "electron-packager . --overwrite --out builds --platform=linux --arch=x64",
-    "build-electron-mac": "electron-packager . --overwrite --out builds --platform=darwin --arch=x64"
+    "dist": "npm run clean && electron-builder -wlm --x64 --ia32",
```
Comment (angrykoala) [inline package.json]: are you sure the clean script works?

File: package.json [hunk 4]
```diff
@@ -1,37 +1,65 @@
...
+  },
+  "devDependencies": {
+    "electron-packager": "^8.5.0",
+    "jshint": "^2.9.4"
+  },
   "scripts": {
     "start": "electron .",
     "test": "echo \"Error: no test specified\" && exit 1",
     "jshint": "jshint index.js app/*.js",
     "build-electron": "electron-packager . --overwrite --out builds --ignore='\\.gitignore|\\.jshintrc'",
```
Comment (angrykoala) [inline package.json]: this may be no longer needed as you use electron builder instead

File: package.json [hunk 5]
```diff
@@ -1,37 +1,65 @@
...
+  "homepage": "https://github.com/angrykoala/mustard#readme",
+  "dependencies": {
+    "browser-id3-writer": "^3.0.2",
+    "electron": "^1.4.14",
+    "jquery": "^3.1.1",
+    "musicmetadata": "^2.0.5",
+    "vue": "^2.1.8"
+  },
+  "devDependencies": {
+    "electron-packager": "^8.5.0",
```
Comment (angrykoala) [inline package.json]: shouldnt we need electron builder here?

**Review by angrykoala — CHANGES_REQUESTED** [review]
some changes in package json

File: package.json (hunk 4 above)
Comment (aledomu) [reply package.json]: I thought it was for testing purposes, although it isn't implemented yet.

File: package.json (hunk 3 above)
Comment (aledomu) [reply package.json]: Oh, that was a dumb mistake.

File: package.json (hunk 4 above)
Comment (angrykoala) [reply package.json]: nope, that's why I'm telling you we don't need it anymore :smile:

File: package.json (hunk 5 above)
Comment (aledomu) [reply package.json]: Right, will fix it.

File: package.json (hunk 4 above)
Comment (aledomu) [reply package.json]: Ah ok

File: package.json [hunk 6]
```diff
@@ -1,37 +1,58 @@
...
+    "vue": "^2.1.8"
+  },
+  "devDependencies": {
+    "electron": "^1.4.14",
+    "electron-builder": "^19.41.1",
+  },
   "scripts": {
     "start": "electron .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "jshint": "jshint index.js app/*.js",
```
Comment (angrykoala) [inline package.json]: Why did you remove jshint?

File: package.json [hunk 7]
```diff
@@ -1,37 +1,58 @@
...
-    "build-electron": "electron-packager . --overwrite --out builds --ignore='\\.gitignore|\\.jshintrc'",
-    "build-electron-all": "npm run build-electron-win && npm run build-electron-linux && npm run build-electron-mac",
-    "build-electron-win": "electron-packager . --overwrite --out builds --platform=win32 --arch=x64",
-    "build-electron-linux": "electron-packager . --overwrite --out builds --platform=linux --arch=x64",
-    "build-electron-mac": "electron-packager . --overwrite --out builds --platform=darwin --arch=x64"
+    "clean": "rm -rf dist"
```
Comment (angrykoala) [inline package.json]: missing , at the end

**Review by angrykoala — CHANGES_REQUESTED** [review]
I found a couple of issues in your changes

File: package.json (hunk 7 above)
Comment (aledomu) [reply package.json]: Done

File: package.json (hunk 6 above)
Comment (aledomu) [reply package.json]: Misunderstood one of your previous questions. Already fixed
